src/Sentry/Extensibility/HubAdapter.cs

```diff
@@ -75,8 +75,8 @@ public void WithScope(Action<Scope> scopeCallback)
         /// Forwards the call to <see cref="SentrySdk"/>.
         /// </summary>
         [DebuggerStepThrough]
-        public Transaction CreateTransaction(string name, string operation)
-            => SentrySdk.CreateTransaction(name, operation);
+        public Transaction StartTransaction()
```

Comment:
Note to myself: check if creating transaction without a name is ok

Reply:
I don't believe Sentry is OK with a transaction without name. Does it make sense to have one? How do we group them in the UI?

Reply:
transaction without name cannot be used by product at all